### PR TITLE
Release 1.20.0

### PR DESCRIPTION
**Pull request by katspaugh**
## Features
* Native WalletConnect (#2535)
Screenshot 2023-10-23 at 16 24 03

## Improvements & bug fixes
* feat: track user's wallet type through user properties (#2688)
* Fix: show nonce warning when above current nonce (#2625)
* fix: after-install in Docker (#2700)
* Fix: uncaught exception in ApprovalEditor (#2674)
* fix: clear awaiting confirmation notifications (#2588)
* fix: Track transaction item open events (#2602)
* fix: Adjust safe creation event to include safeAddress (#2648)
* Update nextjs to v13.5 (#2539)

## Tests & chores
* Tests: Migrate safe apps tests to Sepolia (#2680)
* Tests: Migrate Smoke tests to Sepolia (#2670)
* Tests: Add Assets tests (#2624)
* Tests: Add create safe tests (#2611)
* Tests: Add Remove owner tests (#2587)
* Chore: enable unit test coverage comment (#2637)
* chore: mock data builder for tests (#2675)
* Refactor: map props to hooks (#2564)
* fix: console warnings in owner list (#2612)
* Fix: use local ExplorerButton component (#2685)
* Fix: key prop in HumanDescription (#2698)

### CODE REVIEW

**Comment by liliya-soroka** [comment]
Small UI bug in the "How do I interact with dapps - wrong network is displayed in how to
Steps:
1. open app
2. switch the network using drop-down in the right corner
3. open how do I interact ... 
Current result: The network that was used during the app opening is displayed in the instruction

4. refresh the page manually and check the network in the how to ...
Current result: the correct network is displayed

image

**Comment by liliya-soroka** [comment]
2.  Enable notification did not appear for me when I added my first safe in the PR ( clear install- first opening for the safe). Have we removed it ?
image

**Comment by liliya-soroka** [comment]
2. Connect to Opensea doesn't work on Arbitrum
Steps:
1. try to connect safe to the Opensea on Arbitrum
Current result: Either the error is displayed on opensea side or the QR code dialog is not closed.
Connection appears on safe side
Could we check if it's not our side issue?

File: src/components/walletconnect/WcHints/index.tsx [hunk 1]
```diff
@@ -88,10 +84,6 @@ const WcHints = (): ReactElement => {
     trackEvent(WALLETCONNECT_EVENTS.HINTS_EXPAND)
   }
 
-  if (chain?.chainName) {
-    InteractionSteps[1] = InteractionSteps[1].replace(/%%chain%%/, chain?.chainName)
```
Comment (katspaugh) [inline src/components/walletconnect/WcHints/index.tsx]: @iamacook FYI. You should never modify constants like this.

**Pull request merged**